src/app/components/pages/TeacherRequest.tsx

```diff
@@ -124,9 +103,22 @@ export const TeacherRequest = () => {
                   </Col>
                 </Row>
               )}
+              {isTeacherPending(user) && !messageSent && (
+                <Row data-testid="teacher-pending-account">
+                  <Col className="text-center pt-3">
+                    <span className="h3">You already have a teacher upgrade request pending</span>
+                    <p className="mt-3">
+                      {
```

Comment:
There is no need to evaluate a JS expression when all you have inside is just a string, so the curly braces should be removed.